colossalai/fx/_compatibility.py

```diff
@@ -0,0 +1,42 @@
+from typing import Callable
+
+import torch
+
+from . import META_COMPATIBILITY
+
+
+def compatibility(is_backward_compatible: bool = False) -> Callable:
```

Comment:
I think pytorch already has this API?

Reply:
This is for different uses. Pytorch uses compatibility only for its docstring. I use this to replace incompatible imports with `RuntimeError`. Since PyTorch doesn't have codes for torch12 in torch11, it doesn't care about this. But here in Colossal-AI, we definitely need this version control.